package.json

```diff
@@ -1,13 +1,16 @@
 {
   "name": "films",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "private": true,
   "dependencies": {
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-infinite-scroll": "^0.1.5",
```

Comment:
you're using only react-infinite-scroller, I think you need only one dependency of those